packages/contracts-bedrock/scripts/upgrades/holocene/.env.example

```
@@ -53,6 +53,16 @@ USE_FAULT_PROOFS=true
 # Cannot be 'true' if `USE_FAULT_PROOFS` is 'false'
 USE_PERMISSIONLESS_FAULT_PROOFS=true
 
+# op-contracts release to use for deployments and used for retrieving contract deployment addresses
+# from the superchain-registry.
+#
+# An entry for this release has to exist in file
+# https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-sepolia.toml
+# for Sepolia or
+# https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml
+# for Mainnet.
+OP_CONTRACTS_RELEASE="v1.8.0-rc.3"
```

Just when we're about to merge this PR and tag the release, can we bump this RC counter (also in the SCR).
```suggestion
OP_CONTRACTS_RELEASE="v1.8.0-rc.4"
```
Leave this comment open until that moment.